keps/sig-auth/20190607-certificates-api.md

```diff
@@ -132,6 +133,31 @@ A typical successful issuance proceeds as follows.
    the `.Status.Certificate` field.
 1. The requestor observes the update, and stores the certificate locally.
 
+### Signers
+CSRs have a `signerName` field which is used to specify which signer the CSR creator wants to sign the certificate.
+To support migration from v1beta1 to v1, this required field will be defaulted in v1beta1, but not in v1:
```

Comment:
Also, add fieldSelector support for spec.signerName and specify the required format for validation (qualified namespaced name seems reasonable)

Reply:
The current implementation in #86476 does not specify any restrictions on what the `signerName` value looks like (i.e. qualified namespaced name) - is this something that should be included?